test/geometries/boost_array_as_point.cpp

```diff
@@ -31,6 +35,12 @@ int test_main(int, char* [])
     std::clog << bg::distance(p1, p2) << std::endl;
     std::clog << bg::distance(p2, p3) << std::endl;
     std::clog << bg::distance(p3, p4) << std::endl;
+
+#ifndef BOOST_NO_CXX11_HDR_ARRAY 
+    std::array<double, 3> p5{13,14,15};
+    std::clog << bg::distance(p4, p5) << std::endl;
+#endif //BOOST_NO_CXX11_HDR_ARRAY 
```

Comment:
Nice. This is not about your change, but we should update this test. Writing to clog is not really a unit test, apparently this was never really finished. Anyway, that is not an objective for merging.